google-cloud-bigquery/lib/google/cloud/bigquery/service.rb

```diff
@@ -473,6 +473,7 @@ def query_config query, options = {}
         def resolve_legacy_sql legacy_sql, standard_sql
```

Comment:
Well I finally felt pain when fixing the `resolve_legacy_sql` precedence. These methods have been moved to the Convert module.